src/osd/OSD.cc

```diff
@@ -1417,7 +1418,7 @@ void OSDService::send_incremental_map(epoch_t since, Connection *con,
       // just send latest full map
       MOSDMap *m = new MOSDMap(monc->get_fsid(),
 			       osdmap->get_encoding_features());
-      m->oldest_map = max_oldest_map;
+      m->cluster_osdmap_trim_lower_bound = sblock.cluster_osdmap_trim_lower_bound;
```

Comment:
I didn't check, but only the first bit of this line should be in the rename commit.  The rename commit should only have the m->m->oldest_map -> m->cluster_osdmap_trim_lower_bound bit.  The change from '= sblock.oldest_map' to '= sblock.cluster_osdmap_trim_lower_bound' should be in the commit introducing sblock.cluster_osdmap_trim_lower_bound.  It's annoying, but rename commits really shouldn't change behavior at all.

Reply:
> m->oldest_map -> m->cluster_osdmap_trim_lower_bound bit

* 721c9aea1869bf95a167b8879756b4f71fbf41bf

>   '= sblock.oldest_map' to '= sblock.cluster_osdmap_trim_lower_bound'

* ff58b81ddff5e99a57ca6cf2c5dae560fbc8adad